### PR TITLE
Implemented addProfessor and professorFullName methods in the Section…

### PR DESCRIPTION
**Pull request by Brandon-CSUN**
In the Person abstract class two return methods were implemented and removed getters and settlers from the protected fields instructed by @chizuo. getId and getName methods were implemented because Professor and Student class inherit from the Person class which contains protected fields. This means they won't be able to reference those fields that are set to protect. 

getId returns a Student or Professor ID depending on who is calling on the method. If a student calls on the method, the method going to return the student id. If a professor calls on the method, the method going to return the professor id.
 
getName return method has an if condition where if the student or professor has a middle name not set to an empty string then return the first name, middle name, and last name. if the middle name is set to an empty string then return the first name and last name.  

In the Section class, two methods were implemented addProfessor and profressorFullName. 

The protected Professor professor global field was added to the Section class. 

Added Professor professor optional parameter and set to null.

addProfessor method has an object parameter that gets past a professor object to be set to the protected Professor professor object field for the section and returns true. 

profFullName return method has an if condition where if the protected Professor object field is not equal to null then return the professor full name by invoking the getName method from Person class which will return the professor name. if the protected Professor object field is set to null then return "Staff" since there's no professor assigned to the section.

### CODE REVIEW

File: Models/Section.cs [hunk 1]
```diff
@@ -77,7 +81,19 @@ public bool addStudent(Student student, int addNumber)
...
             }
         }
-        
+
+        public bool addProfossor(Professor prof)
+        {
+            professor = prof;
+            return true;
+        }
+        public string professorFullName()
```
Comment (Jacob-Asaad) [inline Models/Section.cs:90]: Since professorFullName is returning the name of a professor, I feel it should be called getProfessorFullName

Comment (Brandon-CSUN) [reply Models/Section.cs:90]: Okay I make the changes right now

File: Models/Section.cs [hunk 2]
```diff
@@ -77,7 +81,19 @@ public bool addStudent(Student student, int addNumber)
                 return false;
             }
         }
-        
+
+        public bool addProfossor(Professor prof)
```
Comment (chizuo) [inline Models/Section.cs:85]: Professor in the method addProfossor is spelled wrong :P

**Review by chizuo — CHANGES_REQUESTED** [review]
Looks good, but requesting a nitpicky change.

**Review by chizuo — APPROVED** [review]

**Pull request merged**